core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/speech/SpeechRecognizer.java

```
@@ -128,21 +117,24 @@ public void update(LineEvent event) {
 	 *            Handler which will handle the input
 	 * @param ais
 	 *            set custom AudioInputStream.
+	 * @param voiceOutput
+	 *            true if the TTS shall voice the Answer
 	 */
 	public SpeechRecognizer(AudioUserInteraction audioUI, Grammar grammar, SpeechInputHandler inputHandler,
-			AudioInputStream ais) {
+			AudioInputStream ais, boolean voiceOutput) {
 		this.audioUI = audioUI;
 		this.grammar = grammar;
 		this.inputHandler = inputHandler;
 		this.ais = ais;
+		this.voiceOutput = voiceOutput;
 
 		this.tts = TextToSpeech.getTTS();
 
 		// Create the Recognizer
 		try {
 			SpeechRecognizer.this.recognizer = new StreamSpeechRecognizer(this.createConfiguration());
 		} catch (IOException e) {
-			throw new RuntimeException(e);
+			this.logger.error("StreamRecognizer can't be instantiated", e);
```

throw a RuntimeException instead of just log the IOException